tests/samplesheets/pairedEnd.yml

```diff
@@ -0,0 +1,21 @@
+samples:
```

Comment:
These yaml files can be removed I think.
It is a legacy format, and it is not that easy to work with in my opinion. If you think they are easier than the CSV files you can keep them in.